### ISSUE TITLE
Cannot run nmap on CM 13.0 (armv7l; Nexus 5)

### ISSUE THREAD

**Issue by mcepl** [kost/NetworkMapper#16]
When trying to run nmap on the above mentioned system, I get this error (NetworkMapper 1.7 from https://f-droid.org/repository/browse/?fdid=org.kost.nmap.android.networkmapper):

```
Executing:/data/user/0/org.kost.nmap.android.networkmapper/bin/nmap   hubmaier.ceplovi.cz
CANNOT LINK EXECUTABLE: cannot find verneed/verdef for version index=2 referenced by symbol "free" at "/data/user/0/org.kost.nmap.android.networkmapper/bin/nmap"
```

**Comment by ghost** [comment]
I too have encountered this error. 

I have added screensshots and my About phone screen to make it more clear.

![alt text](http://i.imgur.com/uEvqQs3.png)

![alt text](http://i.imgur.com/lPtcKTz.png)

**Comment by kost** [comment]
'm working on it and it seems it is working. Could you test it and let me know?

You need to go to the Settings and Change URL to the following (for the test):
https://github.com/kost/nmap-android/releases/download/v7.25BETA2and1

I think this URL should work as well:
https://github.com/kost/nmap-android/releases/download/v7.25BETA2and4

If you think you can help me testing, I have bunch of release URLs here:
https://github.com/kost/nmap-android/releases

It would be helpful for me to tell me which version works for you.

Thanks in advance!

**Comment by ghost** [comment]
Yes it works sir!  Could you please update the default download URLs. Or maybe map a specific binary for a specific android version instead of being general about it.

**Comment by mcepl** [comment]
Agree, works for me as well.
